__tests__/app.test.js

```diff
@@ -21,3 +27,44 @@ describe("CORE: GET /api/topics", () => {
       });
   });
 });
+
+describe("CORE: GET /api/articles/:article_id", () => {
+  it("responds with article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        [body] = body;
```

Comment:
You should be sending back your responses in an object with a key. i.e. `article`